Tasks/AndroidSigning/task.json

```diff
@@ -38,25 +38,26 @@
             "label": "APK Files",
             "defaultValue": "",
             "required": true,
-            "helpMarkDown": "Relative path from repo root of the files(s).  Wildcards can be used.  For example, `**\\bin\\*.apk` for all APK files in bin subfolder."
+            "helpMarkDown": "Relative path from the repo root to the APK(s) you want to sign. You can use wildcards to specify multiple files. For example, `**\\bin\\*.apk` for all .APK files in the 'bin' subfolder."
```

Comment:
We recommend using / in all file paths. The help doc should reflect the same '**/bin/*.apk'